integration-tests/common.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
```

Comment:
Created the repo, should be just fine IIRC - https://github.com/fabric8-analytics/fabric8-analytics.github.io feel free to experiment with it and push whatever is needed for you.